packages/webapp/src/components/ui/Persona/index.tsx

```diff
@@ -60,7 +60,7 @@ export const Persona: StandardFC = memo(function Persona({ className }) {
 
 		contextMenuItems.push({
 			key: 'toggleOffline',
-			text: `${isOffline ? 'Disable' : 'Enable'} Offline Mode`,
+			text: 'Test Offline Mode',
```

Comment:
We still want to have the label be `Disable` when in offline mode.

Also seems like this should be using a localized string, so we would need to go through the translation hook (which seems to have been an existing issue).

Reply:
Changes made